src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -688,7 +690,7 @@ export default class DefaultAudioVideoController implements AudioVideoController
     }
     if (status.isFailure() || status.isTerminal()) {
       if (this.meetingSessionContext.reconnectController) {
-        return this.reconnect(status);
+        return this.reconnect(status, errorMessage);
```

Comment:
https://github.com/aws/amazon-chime-sdk-js/blob/a4ee269d18b2f796494b38b436e467ebf9ca5fd3/src/audiovideocontroller/DefaultAudioVideoController.ts#L678-L682

- How about removing this warning above and logging it like the following?

```
if (status.isFailure() || status.isTerminal()) {
  if (this.meetingSessionContext.reconnectController) {
    const willRetry = this.reconnect(status);
    if (willRetry) {
      this.logger.warn(`will retry due to status code ${MeetingSessionStatusCode[status.statusCode()]}${error ? ` and error: ${error.message}` : ``}`);
    } else {
      this.logger.error(`failed with status code ${MeetingSessionStatusCode[status.statusCode()]}${error ? ` and error: ${error.message}` : ``}`);
    }
    return willRetry;
  }
}
```

Reply:
Yep, removed the status.isFailure() warning and moved there in the new revision